store/src/main/java/com/book/store/config/SecurityConfig.java

```diff
@@ -31,8 +29,11 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .addFilterBefore(new JwtAuthenticationFilter(jwtUtil, userDetailsService), UsernamePasswordAuthenticationFilter.class)
                 .authorizeHttpRequests(authorizeRequests ->
                         authorizeRequests
-                                .requestMatchers("/auth/**", "/swagger-ui/**")
-                                .permitAll()
+                                .requestMatchers(
+                                        "/auth/**",
+                                        "/swagger-ui/**"
```

Comment:
permit `/error`. 

User should have access to error page in case of exceptions.